Sources/Subprocess/Errors.swift

```diff
@@ -45,3 +46,34 @@ public enum SubprocessError: Error {
     /// Output string could not be encoded
     case outputStringEncodingError
 }
+
+extension SubprocessError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .exitedWithNonZeroStatus(_, let errorMessage):
+            return "\(errorMessage)"
+        case .unexpectedPropertyListObject:
+            return "The property list object could not be cast to expected type"
+        case .unexpectedJSONObject:
+            return "The JSON object could not be cast to expected type"
+        case .inputStringEncodingError:
+            return "Input string could not be encoded"
+        case .outputStringEncodingError:
+            return "Output string could not be encoded"
+        }
+    }
+}
+
+/// Gets us the NSError methods for when used by Objective-C
```

Comment:
Just noticed this minor grammar problem `...methods for when used by...`

```suggestion
/// Common NSError methods for better interop with Objective-C
```